sdf/final-project/presentation.md

```diff
@@ -1,82 +1,49 @@
 # SDF Final Project Rubric - Presentation
-
 This is intended to be filled out by staff members of DPI Tech Apprenticeship program.
 
-Date/Time:
-
-Trainee Name:
-
-Project Name:
-
-Reviewer Name:
-
-## Usefulness
-[ ] Who is this for?
-
-[ ] Is the target user clearly identified somewhere in the presentation?
-
-[ ] Could you imagine someone using this?
+- Date/Time:
+- Trainee Name:
+- Project Name:
+- Reviewer Name:
 
-[ ] Should have some evidence of market research or validation of the problem being solved.
-
-[ ] Is this a compelling/exciting solution?
+<!-- Maybe change this to "story"? -->
+<!-- TODO: is there clear business value? Is it clear how this solution is preparing you for on the job?  (eg ties to a category @ rfp.dpi.dev) -->
+## Business Value
+- [ ] Who is this for? Is the target user clearly identified somewhere in the presentation?
+- [ ] Is there evidence of market research or validation of the problem being solved? (eg talking to potential users)
+- [ ] Is this a compelling/exciting solution? Could you imagine someone using this?
+<!-- TODO: 2 more -->
 
 ### Score (/5):
 
 ### Notes:
 
 ## Slides
-[ ] Is the deck clear and easily understood?
-
-[ ] Deck should be easily understood by the general public
-
-[ ] Deck may include screencasts
-
-[ ] Text should be minimal
-
-[ ] Bullet points should be concise
+- [ ] Is the deck clear and easily understood by the general public?
+- [ ] Is the text minimal/concise?
+- [ ] Is the font size large enough?
+- [ ] Is there a demo video / screencast / gif in the presentation?
+- [ ] Does the demo video / screencast / gif jump straight into the unique parts of the app? (eg doesn't include basic sign-up/sign-in steps)
 
 ### Score (/5):
 
 ### Notes:
 
 ## Speaking
-[ ] Is the narration easily understood?
-
-[ ] Narration should be concise and consistent with the slides
-
-[ ] Should adhere to the time limit
-
-[ ] Speaking should be clear and have proper pacing
+- [ ] Is the narration easily understood?
+- [ ] Is the narration concise and consistent with the slides?
+- [ ] Does the speaker adhere to the time limit?
+- [ ] Is the speaking clear with proper pacing?
+<!-- TODO: 1 more? -->
 
```

Comment:
I think we can evaluate the if the presentation follows a clear structure (e.g. intro, problem statement, main features, future goals, closing, etc.). Meaning how the content of the presentation is organized, as opposed to how it is delivered.

Reply:
👍